doc/user-guide/04-scenarios.adoc

```diff
@@ -1331,3 +1331,109 @@ Finished recovering your system. You can explore it under '/mnt/local'.
 
 Finished in 361 seconds.
 ----
+
+== Using ReaR to repair your system
+Instead of using your ReaR image to completely recover your system from bare
+metal (as illustrated in most of the above scenarios), you can also use it
+as a live media to boot a broken but hopefully repairable system.
+
+Once booted on your recovery image, the `repair` workflow will:
+
+* activate all Volume Groups
+
+* offer to decrypt any LUKS-encrypted filesystem that may be present
+
+* mount all the target filesystems (including the most important virtual 
+  ones) below `/mnt/local`
+
+thereby making it possible for you to explore your system at will,
+correcting any configuration mistake that may have prevented its startup, 
+or allowing you to simply `chroot` into it and further repair it using its 
+own administrative tools.
+
+One important point to remember is that the `repair` workflow on its own 
+won't modify the target system in any way. Of course, once the target 
```

Comment:
Now I am wondring if the workflow name 'repair' isn't misleading
because it actually does not 'repair' anything.

Shouldn't it be better named according to what is actually does
like 'mountonly'.

Background info what I have in mind:

I am thinking about how to use this new workflow here
to complement my currently experimental and incomplete
workflows 'restoreonly', 'layoutonly', and 'finalizeonly'.

So if this new workflow basically only mounts the filesystems
it could be used like (assume it is named 'mountonly')
```
# rear mountonly
# rear restoreonly
```
plus optionally
```
# rear finalizeonly
```
to only restore the backup and optionally also
recreate the initrd plus reinstall the bootloader.

When this new workflow is called 'repair'
that use case looks strange:
```
# rear repair
# rear restoreonly
```

Reply:
Workflow renamed to `mountonly` as suggested.